src/Prism.Core/Extensions/TaskExtensions.cs

```diff
@@ -73,11 +73,19 @@ public async static void Await(this Task task, Action completedCallback, Action<
             try
             {
                 await task.ConfigureAwait(configureAwait);
-                completedCallback?.Invoke();
             }
             catch (Exception ex)
             {
                 errorCallback?.Invoke(ex);
+                return;
+            }
+            try
+            {
+                completedCallback?.Invoke();
+            }
+            catch {
```

Comment:
I'm not user I like swallowing the exceptions in the completed callback. If an error happens in the callback, the developer should be notified via an exception. if anything we should probably rethrow the exception

Reply:
If I re-throw the exception, that would be an exception within an async void method that you're not awaiting, so you wouldn't see it, though it might crash the program.

Reply:
I could make it a Task and make the calling methods run the task as a separate thread, but then they're still going to not be able to notify of the error.

Reply:
I'm curious if something like this would be better
```cs
            bool error = false;
            try
            {
                await task.ConfigureAwait(configureAwait);
            }
            catch (Exception ex)
            {
                error = true;
                errorCallback?.Invoke(ex);
            }
            finally
            {
                if (!error)
                    completedCallback?.Invoke();
            }
```

Reply:
@Binderbound did you see my suggestion? Thoughts?